truss/tests/templates/control/control/test_server.py

```diff
@@ -273,14 +259,12 @@ async def test_retries(client, app):
         ]
     )
 
-    with (
-        patch("endpoints.INFERENCE_SERVER_START_WAIT_SECS", new=4),
-        pytest.raises(httpx.RemoteProtocolError),
-    ):
-        await client.get("/v1/models/model")
+    with patch("endpoints.INFERENCE_SERVER_START_WAIT_SECS", new=4):
```

Comment:
Previously the control server would raise an error - now it'll log the error and return a much nicer 500 to the end client